packages/core/src/ui/UIUtils.ts

```
@@ -1,31 +1,42 @@
 import { IUICanvas } from "./IUICanvas";
 
 export class UIUtils {
   private static _renderQueue: RenderQueue;
   private static _virtualCamera: VirtualCamera;
   private static _viewport: Vector4;
+  private static _overlayCamera: OverlayCamera;
 
-  static renderOverlay(engine: Engine, uiCanvases: DisorderedArray<IUICanvas>): void {
+  static renderOverlay(engine: Engine, scene: Scene, uiCanvases: DisorderedArray<IUICanvas>): void {
+    const { canvas, _hardwareRenderer: rhi, _renderContext: renderContext, _batcherManager: batcherManager } = engine;
     const uiRenderQueue = (this._renderQueue ||= new RenderQueue(RenderQueueType.Transparent));
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)